src/layouts/LandingPageLayout.res

```diff
@@ -31,33 +31,35 @@ module PlaygroundHero = {
     {
       res: `module Button = {
   @react.component
-  let make = (~count: int) => {
+  let make = (~count) => {
```

Comment:
Remove type annotation, because it's not needed

Reply:
It's not needed, but depending if the component is a "public" component, I'd always type the interface if possible.

Reply:
I think it might be confusing to have a type annotation that is not actually required, for example, to work.

Reply:
> It's not needed, but depending if the component is a "public" component, I'd always type the interface if possible.

In the .resi 🙂 

I am also in favor of removing the annotation from the example.

Reply:
Oh well, then go wild 🤷